Makefile

```diff
@@ -162,14 +162,27 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+operator-sdk:
+ifeq (,$(shell which operator-sdk 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(OPERATOR-SDK)) ;\
```

Comment:
Where is this OPERATOR _SDK env variable defined? Are we overriding the operator SDK installed locally?
If that is the case, lets not do it, install our version of operator-sdk inside the project and use it from there